org.openkj.OpenKJ.yml

```diff
@@ -0,0 +1,70 @@
+app-id: org.openkj.OpenKJ
+runtime: org.kde.Platform
+runtime-version: 5.15
+sdk: org.kde.Sdk
+command: openkj
+finish-args: 
+  - "--share=ipc"
+  - "--socket=x11"
+  - "--device=dri"
+  - "--share=network"
+    #  - "--socket=wayland"
+  - "--socket=pulseaudio"
+  - "--filesystem=host"
+  - "--env=LADSPA_PATH=/app/lib/ladspa/"
```

Comment:
do you really support LADSPA or is it just cut and paste? If you do you probably want to add support for plugins (Pitivi does it https://github.com/flathub/org.pitivi.Pitivi/blob/master/org.pitivi.Pitivi.json)

Reply:
I use LADSPA solely for the rubberband plugin that gets built in the flatpak. The software uses it for a pitch shifter since sounds better than the soundtouch one that ships as a gstreamer plugin.  There's no support for user specified LADSPA plugins in OpenKJ currently.